torch/csrc/autograd/python_function.cpp

```diff
@@ -806,6 +806,18 @@ PyObject* THPFunction_name(PyObject* self, PyObject* noargs) {
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPFunction_maybe_clear_saved_tensors(
+    PyObject* self,
+    PyObject* noargs) {
+  HANDLE_TH_ERRORS;
+  auto cdata = ((THPFunction*)self)->cdata.lock();
+  if (!cdata->retain_variables) {
```

Comment:
This is a not a thing I'm afraid.
I think you will have to use the same approach as https://github.com/pytorch/pytorch/pull/82544